City-of-Helsinki/drupal-infofinland
# UHF-10449

# [UHF-10449](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10449)
List of links element is missing from pages

## What was done
Enabled all of the paragraphs
(extra: Disabled purge-module)

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10449`
  * `make fresh`
* Run `make drush-cr`
* Setup UI

## How to test
Go to a page with list of links element, for example /fi/moving-to-finland/eu-citizens (under Töihin Suomeen -opas -title)
- The link list element should be visible


[UHF-10449]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
